Packs/GoogleKubernetesEngine/Integrations/GoogleKubernetesEngine/GoogleKubernetesEngine.py

```diff
@@ -7,7 +7,7 @@
 from google.cloud.container_v1 import ClusterManagerClient
 from google.protobuf.json_format import MessageToDict
 from google.protobuf.message import Message
-from google.cloud.container_v1 import enums
+from google.cloud.container_v1 import types
```

Comment:
do we have a `verify.py` file in the dockerfiles repo to check the imports we are using?